lib/client-options.js

```diff
@@ -34,8 +35,10 @@ function defaultOptions () {
     },
     socketOptions: {
       connectTimeout: 5000,
+      defunctReadTimeoutThreshold: 64,
       keepAlive: true,
       keepAliveDelay: 0,
+      readTimeout: 0,
```

Comment:
I think we should default it to 12000, if possible in the next minor version. I created a ticket for it [NODEJS-152](https://datastax-oss.atlassian.net/browse/NODEJS-152).